qa/SidechainTestFramework/scutil.py

```diff
@@ -485,7 +485,7 @@ def initialize_sc_datadir(dirname, n, model, bootstrap_info=SCBootstrapInfo, sc_
     config = tmpConfig % {
         'NODE_NUMBER': n,
         'DIRECTORY': dirname,
-        'LOG_FILE_LEVEL': log_info.logFileLevel,
+        'LOG_FILE_LEVEL': "all",
```

Comment:
revert back the change.
If need you may add execution argument to set a proper log level for both application and/or python logs

Reply:
Sorry I wrongly pushed.